### DISCUSSION TITLE
Unable to SSH

### DISCUSSION THREAD

**Discussion by qfertig** [xblax/flashforge_ad5m_klipper_mod#72] [General]
Hello, 

This may be a noobie user error, however I cannot ssh into my 5m. I have klipper installed, I have found my IP address in windows, and I can also ping it just fine with no errors. When I try to ssh into via putty, it just refuses to connect via the IP. I have even tried to set a static IP address within windows, and yet again, was able to ping it but not able to SSH into it. Any help would be greatly appreciated, thanks!

**Comment by consp** [comment]
> I have found my IP address in windows, and I can also ping it just fine with no errors.

Are you sure you are not connecting to the IP of your windows PC? You need the IP of the printer, not your PC.

**Reply by qfertig** [reply]
Yeah, I got the Ethernet port being used on my USB c adapter when connected and verified by disconnecting the adapter. Did Ip config, grabbed the IP and didn't seen to work. Also tried statically assigning it through windows, and when trying to SSH into it with putty it's not working.

**Reply by consp** [reply]
>  Did Ip config, grabbed the IP and didn't seen to work. 

I'm pretty sure you are trying to ssh to your windows pc. That's not going to work. If the ping is < 1ms then it's your pc, not the printer you are connecting to.

**Reply by qfertig** [reply]
I got it to work, I saw that klipperscreen was only in the alpha, what command do I need to run with SSH to install? Thanks for the help btw.

**Reply by mallison01** [reply]
You do not need to run any command to install. you need to grad the file called:  Adventurer5M-KlipperMod-v00.03-pre-alpha-klipperscreen.tgz (assuming you have a 5M and not the Pro, if you have the pro you need to change it from Adventurer5M- to Adventurer5MPro-.

Drop that file on a usb, plug it in and then turn on the printer and wait, mine took a few minutes to install, it will tell you on the flashforge screen when it is done.

Once installed you can use the screen to connect to your wifi i think or use a wired network which is what i do, so i have not tried wifi yet.

once you have your printers Ip address you can connect to mainsail or fluidd via the ip address x.x.x.x:4000 for mainsail or 4001 for fluidd

[Adventurer5M-KlipperMod-v00.03-pre-alpha-klipperscreen.tgz](https://github.com/xblax/flashforge_adm5_klipper_mod/releases/download/v00.03-beta/Adventurer5M-KlipperMod-v00.03-pre-alpha-klipperscreen.tgz)

**Reply by qfertig** [reply]
Thanks so much! It was the noob part of me that had a hard time finding the IP, I connected to my wifi router and go the IP through that since the IPV4 I was retrieving was weirdly my PC through the USB C hub I own. I will try the klipperscreenThank you very much! I had a hard time finding the IP address at first due to my lack of experience. Eventually, I connected to my Wi-Fi router and retrieved the IP address that way. The IPV4 address I was initially retrieving was actually linked to my PC through the USB C hub I own, which was confusing. I plan on trying the klipperscreen when I get home, but I couldn't find a "how to" guide on it. Although, it looks like it was released as a bundle in the alpha version. However, I am quite new to GitHub and navigating stock Klipper in general.  It was the only thing that I didn't see a "how-to" guide on. I also saw it was released as a bundle in the alpha version. But then again, I am new to github and interacting with stock klipper in general, so thank you for your help!

**Comment by MASalmeron** [comment]
Thank you very much, there is a lot of work in this project!!!

**Comment by Matt-Armfield** [comment]
So is SSH enabled in beta .05?
